heartbeat/galera

```diff
@@ -254,6 +257,7 @@ Cluster check user password
 <action name="monitor" depth="0" timeout="30" interval="20" />
 <action name="monitor" role="Master" depth="0" timeout="30" interval="10" />
 <action name="monitor" role="Slave" depth="0" timeout="30" interval="30" />
+<action name="monitor" role="Slave" depth="0" timeout="30" interval="8" on-fail="block"/>
```

Comment:
Why not name="sync-check"?
Nothing says you need to use monitor for everything :-)